tests/lib/tools/suite/snaps-state/task.yaml

```diff
@@ -0,0 +1,67 @@
+summary: smoke test for the snaps-state tool
+prepare: |
+    apt-get install -y shellcheck
```

Comment:
You need to constraint this via `systems: ` to install packages with apt-get.

Reply:
no shellcheck installed anymore as part of the test.